dvc/repo/ls_url.py

```diff
@@ -0,0 +1,37 @@
+from dvc.exceptions import URLMissingError
+from dvc.fs import get_cloud_fs
+
+
+def parse_external_url(url, config=None):
+    remote_config = dict(config) if config else {}
+    remote_config["url"] = url
+    fs_cls, fs_config, fs_path = get_cloud_fs(None, **remote_config)
+    fs = fs_cls(**fs_config)
+    return fs, fs_path
+
+
+def ls_url(url, *, config=None, recursive=False):
```

Comment:
For the record: discussed that there is an inconsistency here, where use use `config` but in `imp_url` we use `fs_config` as a name.